docs/chroot/README.md

```diff
@@ -0,0 +1,35 @@
+# Deploy Minio on Chrooted Environment [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io) [![Go Report Card](https://goreportcard.com/badge/minio/minio)](https://goreportcard.com/report/minio/minio) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/minio.svg?maxAge=604800)](https://hub.docker.com/r/minio/minio/) [![codecov](https://codecov.io/gh/minio/minio/branch/master/graph/badge.svg)](https://codecov.io/gh/minio/minio)
+
+Chroot allows user based namespace isolation on many standard Linux deployments.
+
+## 1. Prerequisites
+* Familiarity with [chroot](http://man7.org/linux/man-pages/man2/chroot.2.html)
+* Chroot installed on your machine.
+
+## 2. Install Minio in Chroot
+```sh
+mkdir -p /mnt/export/${USER}/bin
+wget https://dl.minio.io/server/minio/release/linux-amd64/minio -O /mnt/export/${USER}/bin/minio
+chmod +x /mnt/export/${USER}/bin/minio
+```
+
+## 3. Run Standalone Minio in Chroot
+### GNU/Linux
+```sh
+chroot --userspec ${USER}:{USER} /mnt/export/${USER} /bin/minio --config-dir=/.minio server /data
+
```

Comment:
We should specify that this command would need to be executed by root. 

Also `s/${USER}:{USER}/${USER}:${USER}`. Alternatively we should specify that after `:` the group name can be given.

Finally, when I tried it, I couldn't get it to work:

```
$ find /tmp/chroot/
/tmp/chroot/
/tmp/chroot/bin
/tmp/chroot/bin/minio
/tmp/chroot/ls
$ sudo chroot --userspec aditya:aditya /tmp/chroot/ /bin/minio --config-dir=/.minio server /data
FATA[0000] Invalid command line arguments server=‘:9000’, args=[/data]  cause=open /proc/mounts: no such file or directory source=[server-main.go:96:serverHandleCmdArgs()]
```